pkg/network/protocols/http2/protocol.go

```diff
@@ -343,6 +353,47 @@ func (p *protocol) GetStats() *protocols.ProtocolStats {
 	}
 }
 
+// UpdateKernelTelemetry should be moved to the HTTP/2 part as well
+func (p *protocol) UpdateKernelTelemetry(mgr *manager.Manager) {
+	var zero uint32
+
+	for {
+		mp, _, err := mgr.GetMap(probes.HTTP2TelemetryMap)
+		if err != nil {
+			log.Warnf("error retrieving http2 telemetry map: %s", err)
+			return
+		}
+
+		http2Telemetry := &HTTP2Telemetry{}
+		if err := mp.Lookup(unsafe.Pointer(&zero), unsafe.Pointer(http2Telemetry)); err != nil {
+			// This can happen if we haven't initialized the telemetry object yet
+			// so let's just use a trace log
+			if log.ShouldLog(seelog.TraceLvl) {
+				log.Tracef("error retrieving the http2 telemetry struct: %s", err)
+			}
```

Comment:
what do you mean? I'm not sure how is it possible

Reply:
changed it to log warning, as this is only telemetry